### ISSUE TITLE
ImportError: cannot import name 'ElectraForMaskedLM'

### ISSUE THREAD

**Issue by Moradnejad** [ThilinaRajapakse/simpletransformers#320]
I get error in this line: `from simpletransformers.classification import ClassificationModel`

Trace back:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-4-01a1c91e766a> in <module>
----> 1 from simpletransformers.classification import ClassificationModel
      2 import pandas as pd

/opt/conda/lib/python3.6/site-packages/simpletransformers/classification/__init__.py in <module>
      1 from simpletransformers.classification.classification_model import ClassificationModel
----> 2 from simpletransformers.classification.multi_label_classification_model import MultiLabelClassificationModel
      3 from simpletransformers.classification.multi_modal_classification_model import MultiModalClassificationModel

/opt/conda/lib/python3.6/site-packages/simpletransformers/classification/multi_label_classification_model.py in <module>
      6 
      7 from simpletransformers.classification import ClassificationModel
----> 8 from simpletransformers.custom_models.models import (
      9     BertForMultiLabelSequenceClassification,
     10     RobertaForMultiLabelSequenceClassification,

/opt/conda/lib/python3.6/site-packages/simpletransformers/custom_models/models.py in <module>
     12 from transformers import FlaubertModel
     13 from transformers.configuration_xlm_roberta import XLMRobertaConfig
---> 14 from transformers import ElectraForMaskedLM, ElectraForPreTraining
     15 from torch.nn import BCEWithLogitsLoss
     16 

ImportError: cannot import name 'ElectraForMaskedLM'
```

**Comment by Moradnejad** [comment]
My code was working in a few days ago

**Comment by remotejob** [comment]
I have completely the same issues.

**Comment by ThilinaRajapakse** [comment]
Duplicate of #306 

Literally the same title. :sweat_smile:

**Comment by mohamedkhanafer** [comment]
Just got the same error too!

**Comment by ThilinaRajapakse** [comment]
Closed as duplicate #306
